pkg/ebpf/tracee.go

```diff
@@ -937,89 +838,41 @@ func (t *Tracee) initBPF() error {
 		BPFObjName:      t.config.BPFObjPath,
 	}
 
+	// Open the eBPF object file (create a new module)
+
```

Comment:
nit: why do we keep these extra lines between the comment and the code relevant to the comment? (here and in many other places)

Reply:
styling preferences. we should start enforcing one, clearly my preferences are different than yours, and different from Nadavs (who does comments like //something). =(